doc/man/zen-cli.1

```diff
@@ -1,9 +1,9 @@
-.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.6.
-.TH ZEN-CLI "1" "September 2022" "zen-cli v3.2.0" "User Commands"
+.\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.49.1.
+.TH ZEN-CLI "1" "January 2023" "zen-cli v3.2.1-679b76dd4" "User Commands"
 .SH NAME
-zen-cli \- manual page for zen-cli v3.2.0
+zen-cli \- manual page for zen-cli v3.2.1-679b76dd4
 .SH DESCRIPTION
-Horizen RPC client version v3.2.0
+Horizen RPC client version v3.2.1\-679b76dd4
```

Comment:
Lets change this to `v3.2.1` without the build number.

Reply:
Addressed with force push (for keeping history clean) in 58c1f119cd7d8eae61fe426a91a562767fd76c70.